pages.ko/common/basename.md

```diff
@@ -5,8 +5,12 @@
 
 - 경로에서 파일 이름만 표시:
 
-`basename {{path/to/file}}`
+`basename {{경로/대상/파일}}`
+
+- 경로에서 가장 오른쪽 디렉터리 이름만 표시:
+
+`basename {{경로/대상/디렉토리/}}`
```

Comment:
```suggestion
`basename {{경로/대상/디렉토리}}`
```

Reply:
`basename {{path/to/directory/}}` in English original
It looks like it needs to be modified.